src/serlo-editor/plugins/text/components/hovering-toolbar-controls.tsx

```diff
@@ -31,10 +33,14 @@ export function HoveringToolbarControls({
             tooltipText={control.title}
             onMouseDown={(event) => {
               event.preventDefault()
+              event.stopPropagation()
               isNestedControlButton(control)
                 ? setSubMenu(index)
                 : control.onClick(editor)
             }}
+            onClick={(event) => {
+              event.stopPropagation()
+            }}
```

Comment:
not sure if both `stopPr…` are needed, did you test is further?
Also a short comment why we need this would help future me :)